src/renderer/domains/network/account/service.ts

```diff
@@ -34,9 +34,12 @@ function uniqId(account: AnyAccountDraft) {
 }
 
 function isCryptoMatch(account: Pick<AnyAccount, 'cryptoType'>, chain: Chain): boolean {
-  const cryptoType = networkUtils.isEthereumBased(chain.options) ? CryptoType.ETHEREUM : CryptoType.SR25519;
+  const isEthereumBased = networkUtils.isEthereumBased(chain.options);
+  if (isEthereumBased) {
```

Comment:
`isEthereumBased` is used only once, can omit this variable
```suggestion
  if (networkUtils.isEthereumBased(chain.options)) {
```